doc/api/addons.markdown

```diff
@@ -85,18 +100,27 @@ to build your module in a JSON-like format. This file gets compiled by
       ]
     }
 
-The next step is to generate the appropriate project build files for the
-current platform. Use `node-gyp configure` for that.
+*Note: A version of the `node-gyp` utility is bundled and distributed with
+Node.js as part of `npm`. This version is not made directly available for
+developers to use and is intended only to support the ability to use the
+`npm install` command to compile and install Addons. Developers who wish to
+use `node-gyp` directly can install it using the command
+`npm install -g node-gyp`*
 
-Now you will have either a `Makefile` (on Unix platforms) or a `vcxproj` file
-(on Windows) in the `build/` directory. Next, invoke the `node-gyp build`
-command.
+Once the `binding.gyp` file has been created, use `node-gyp configure` to
+generate the appropriate project build files for the current platform. This
+will generate either a `Makefile` (on Unix platforms) or a `vcxproj` file
+(on Windows) in the `build/` directory.
 
-Now you have your compiled `.node` bindings file! The compiled bindings end up
-in `build/Release/`.
+Next, invoke the `node-gyp build` command to generate the compiled `addon.node`
+file. This will be put into the `build/Release/` directory.
 
-You can now use the binary addon in a Node.js project `hello.js` by pointing
-`require` to the recently built `hello.node` module:
+When using `npm install` to install a Node.js Addon, npm uses it's own bundled
+version of `node-gyp` to perform this same set of actions, generating a
+compiled version of the Addon for the user's platform on demand.
+
+Once built, the binary Addon can be used from within Node.js by pointing
+`require` to the built `addon.node` module:
 
     // hello.js
     const addon = require('./build/Release/addon');
```

Comment:
Let's add something like this:

> As the exact path to the `addon.node` file may vary depending on the compile type (i.e. sometimes it may be in `./build/Debug/`), it is generally recommended that addons use the [bindings](https://github.com/TooTallNate/node-bindings) package to load the compiled module.